vm/rust/versioned_constants_13_2.json

```diff
@@ -0,0 +1,608 @@
+{
+  "tx_event_limits": {
+    "max_data_length": 300,
+    "max_keys_length": 50,
+    "max_n_emitted_events": 1000
+  },
+  "gateway": {
+    "max_calldata_length": 5000,
+    "max_contract_bytecode_size": 81920
+  },
+  "invoke_tx_max_n_steps": 10000000,
+  "l2_resource_gas_costs": {
+    "gas_per_data_felt": [
+      128,
+      1000
+    ],
+    "event_key_factor": [
+      2,
+      1
+    ],
+    "gas_per_code_byte": [
+      875,
+      1000
+    ]
+  },
+  "disable_cairo0_redeclaration": true,
+  "max_recursion_depth": 50,
+  "segment_arena_cells": false,
+  "os_constants": {
+    "block_hash_contract_address": 1,
+    "call_contract_gas_cost": {
+      "entry_point_gas_cost": 1,
+      "step_gas_cost": 10,
+      "syscall_base_gas_cost": 1
+    },
+    "constructor_entry_point_selector": "0x28ffe4ff0f226a9107253e17a904099aa4f63a02a5621de0576e5aa71bc5194",
+    "default_entry_point_selector": 0,
+    "deploy_gas_cost": {
+      "entry_point_gas_cost": 1,
+      "step_gas_cost": 200,
+      "syscall_base_gas_cost": 1
+    },
+    "emit_event_gas_cost": {
+      "step_gas_cost": 10,
+      "syscall_base_gas_cost": 1
+    },
+    "entry_point_gas_cost": {
+      "entry_point_initial_budget": 1,
+      "step_gas_cost": 500
+    },
+    "entry_point_initial_budget": {
+      "step_gas_cost": 100
+    },
+    "entry_point_type_constructor": 2,
+    "entry_point_type_external": 0,
+    "entry_point_type_l1_handler": 1,
+    "error_block_number_out_of_range": "Block number out of range",
+    "error_invalid_input_len": "Invalid input length",
+    "error_invalid_argument": "Invalid argument",
+    "error_out_of_gas": "Out of gas",
+    "execute_entry_point_selector": "0x15d40a3d6ca2ac30f4031e42be28da9b056fef9bb7357ac5e85627ee876e5ad",
+    "fee_transfer_gas_cost": {
+      "entry_point_gas_cost": 1,
+      "step_gas_cost": 100
+    },
+    "get_block_hash_gas_cost": {
+      "step_gas_cost": 50,
+      "syscall_base_gas_cost": 1
+    },
+    "get_execution_info_gas_cost": {
+      "step_gas_cost": 10,
+      "syscall_base_gas_cost": 1
+    },
+    "initial_gas_cost": {
+      "step_gas_cost": 100000000
+    },
+    "keccak_gas_cost": {
+      "syscall_base_gas_cost": 1
+    },
+    "keccak_round_cost_gas_cost": 180000,
+    "l1_gas": "L1_GAS",
+    "l1_gas_index": 0,
+    "l1_handler_version": 0,
+    "l2_gas": "L2_GAS",
+    "l2_gas_index": 1,
+    "library_call_gas_cost": {
+      "call_contract_gas_cost": 1
+    },
+    "sha256_process_block_gas_cost": {
+      "step_gas_cost": 1852,
+      "range_check_gas_cost": 65,
+      "bitwise_builtin_gas_cost": 1115,
+      "syscall_base_gas_cost": 1
+    },
+    "memory_hole_gas_cost": 10,
+    "nop_entry_point_offset": -1,
+    "range_check_gas_cost": 70,
+    "bitwise_builtin_gas_cost": 594,
```

Comment:
I think you're missing some other fields
"memory_hole_gas_cost": 10,
        "nop_entry_point_offset": -1,
        "range_check_gas_cost": 70,
        "pedersen_gas_cost": 0,
        "bitwise_builtin_gas_cost": 594,
        "ecop_gas_cost": 0,
        "poseidon_gas_cost": 0,
        "add_mod_gas_cost": 0,
        "mul_mod_gas_cost": 0,

        https://github.com/starkware-libs/sequencer/blob/blockifier-v0.8.0-rc.3/crates/blockifier/resources/versioned_constants_13_2.json